content/millennium/r4/encounters/encounter.md

```diff
@@ -61,6 +62,7 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  ID                                                        | Value\[x] Type                                                   | Description
 -----------------------------------------------------------|------------------------------------------------------------------|----------------------------------------------------------------
  `client-organization`                                     | [`Reference`](https://hl7.org/fhir/r4/references.html#Reference) | The financially responsible organization.
+ `custom-attribute`                                        | None (contains nested extensions)                                | A client defined custom attribute for the resource.
```

Comment:
is this what we said for other complex extensions?

Reply:
This is fine. We've done it like this for other complex extensions.

Reply:
We are basing a good portion of this documentation off of the [financial-transaction-allocation](https://fhir.cerner.com/millennium/r4/financial/financial-transaction/#custom-extensions) documentation, which denotes sub-extensions as 'nested extensions'